src/test/java/gov/cabinetoffice/gap/adminbackend/services/SnsServiceTest.java

```diff
@@ -23,6 +23,10 @@ public class SnsServiceTest {
 
     private static SnsConfigProperties snsConfigProperties;
 
+    private final String MESSAGE_ID = "mockMessageId";
```

Comment:
A bit nitpicky as it's a test class but these should be static.

`private static final String MESSAGE_ID = "mockMessageId";`
` private static final String ERROR = "error publishing message";`